providers/gladly.go

```diff
@@ -0,0 +1,33 @@
+package providers
+
+const Gladly = "gladly"
+
+func init() {
+	SetInfo(Gladly, ProviderInfo{
+		DisplayName: "Gladly",
+		AuthType:    Basic,
+		BaseURL:     "https://{{.workspace}}/api",
```

Comment:
https://developer.gladly.com/rest/#section/Overview/REST-API

I think the base url is wrong. Please fix this.

Reply:
Withampersand's domain is partner-withampersand.us-uat.gladly.qa, when I tried to make API calls via partner-withampersand.us-uat.gladly.com, I get an error, when changing it to .gladly.qa, the call is successful. I have attached the screenshots
![Screenshot 2567-08-20 at 15 40 27](https://github.com/user-attachments/assets/74b6539f-ed55-4ca7-80a3-f6a4fcbaa9e3)
![Screenshot 2567-08-20 at 15 41 14](https://github.com/user-attachments/assets/63fb5b0a-7794-442e-a4d0-fdd368de0c72)

Reply:
I see where this is coming from thanks for the screenshot. But according to the doc https://developer.gladly.com/rest/#section/Overview/Testing

`.qa` is for testing sandbox environment and real accounts will be using `gladly.com`

I think the correct approach should be to use `gladly.com`. You can change this to `gladly.com` and I can accept this without testing `gladly.com` knowing the limitation. 

@laurenzlong what do you think? I think we can support QA testing environment later when we support modules.

Reply:
@Beadko as you change this to `gladly.com` you can add a comment in code that this does not support `.qa` yet.

Reply:
@Beadko Lauren and I had discussion on this, and decided that we should have separate connectors for qa environment and prod environment. Can you update this PR to have both connectors here?